src/webgpu/util/conversion.ts

```diff
@@ -564,7 +564,7 @@ export class Scalar {
 
 /** Create an f64 from a numeric value, a JS `number`. */
 export function f64(value: number): Scalar {
-  const arr = new Float32Array([value]);
+  const arr = new Float64Array([value]);
```

Comment:
Probably worth a mention in the CL description, or splitting out to a separate change.

Reply:
Done